\documentclass[10pt]{article}
\usepackage{multirow}
\usepackage{hyperref}
\usepackage{xr}
\usepackage[export]{adjustbox}
\usepackage{float}
\usepackage[caption = false]{subfig}
\usepackage{graphicx}

%\externaldocument{supplement}

\topmargin 0.0cm
\oddsidemargin 0.2cm
\textwidth 16cm 
\textheight 21cm
\footskip 1.0cm


%The next command sets up an environment for the abstract to your paper.

\newenvironment{sciabstract}{%
\begin{quote} \bf}
{\end{quote}}


% If your reference list includes text notes as well as references,
% include the following line; otherwise, comment it out.

\renewcommand\refname{References and Notes}

% The following lines set up an environment for the last note in the
% reference list, which commonly includes acknowledgments of funding,
% help, etc.  It's intended for users of BibTeX or the {thebibliography}
% environment.  Users who are hand-coding their references at the end
% using a list environment such as {enumerate} can simply add another
% item at the end, and it will be numbered automatically.

\newcounter{lastnote}
\newenvironment{scilastnote}{%
\setcounter{lastnote}{\value{enumiv}}%
\addtocounter{lastnote}{+1}%
\begin{list}%
{\arabic{lastnote}.}
{\setlength{\leftmargin}{.22in}}
{\setlength{\labelsep}{.5em}}}
{\end{list}}


% Include your paper's title here

\title{ViFi: Viral integration and mRNA fusion identification} 


% Place the author information here.  Please hand-code the contact
% information and notecalls; do *not* use \footnote commands.  Let the
% author contact information appear immediately below the author names
% as shown.  We would also prefer that you don't change the type-size
% settings shown here.

\author
{Nam-phuong Nguyen$^{1}$, Viraj Deshpande$^{1}$, 
Jens Luebeck$^{2}$, Paul Mischel$^{3}$, and Vineet Bafna$^{1\ast}$\\
\normalsize{$^{1}$Computer Science and Engineering, University of California,}\\
\normalsize{San Diego, La Jolla, 92093, USA.,}\\
\normalsize{$^{2}$Bioinformatics and Systems Biology Program, University of California,}\\
\normalsize{San Diego, La Jolla, 92093, USA.,}\\
\normalsize{$^{3}$Department of Pathology, University of California,}\\
\normalsize{San Diego, La Jolla, 92093, USA.,}\\
\normalsize{$^\ast$To whom correspondence should be addressed; E-mail:  vbafna@eng.ucsd.edu.}
}

% Include the date command, but leave its argument blank.

\date{}

%%%%%%%%%%%%%%%%% END OF PREAMBLE %%%%%%%%%%%%%%%%



\begin{document} 

% Double-space the manuscript.

\baselineskip24pt

% Make the title.

\maketitle 

% Place your abstract within the special {sciabstract} environment.

\begin{sciabstract}
An estimated 12\% of all human cancers are caused by oncoviruses, with more than 80\% of the cases occurring in developing countries.  In cancers such as cervical cancer and hepatocellular carcinoma, the integration of the viral genome into the host cell's genome is an important factor tumorigenesis.  Standard approaches to viral integration detection is identifying paired-end reads mapping to both the human and viral references.  However, read-based reference mapping has difficulty identifying integration sites from rapidly evolving viruses or from novel virus strains.  We present ViFi, a viral integration detection tool which uses both reference-based read mapping and a phylogenetic-based approach to identify viral reads.  Key to this framework is the use of profile Hidden Markov models to represent the viral family of interest, allowing the detection of highly mutated or novel  sequences belonging to a viral family.  In addition, ViFi uses mappability scores to reduce false positive detections.  ViFi can be run on WGS and RNA-seq data, allowing for both the detection of integration sites and fusion transcripts.  We compare Viral-HMM to VERSE and ViralFusionSeq on both simulated NGS data and real biological data with experimentally verified integration sites.  Our results show that ViFi is more sensitive in detecting integration sites and has a lower false positive detection rate.
\end{sciabstract}

\section*{Introduction}
Viral infection and integration plays a key role in tumorigenesis, however, determining the causative factors has been challenging.  Difficulties include the long latency period between infection and cancer formation (between 15 to 40 years for some cancers), the viruses that cause cancer in some patients often do not cause cancer in many other patients, and some viruses are indirect carcinogens (i.e., viral genes are not present in the tumor cells)~\cite{Hausen2009}.  Furthermore the human cancer viruses span across a diverse range of viral families, including retroviruses, positive-stranded RNA viruses, and small and large DNA viruses.  While these viral families have different cellular targets and different methods for proliferation, one key insight is that some of these viruses (Human papillomavirus (HPV), Molluscum contagiosum virus (MCV), Hepatitis B virus (HBV), and Human T-lymphotropic virus (HTLV)) have mechanisms for integrating directly into the host genome, and a tumor sample will show the same integration site throughout the sample, suggesting that integration occurred before clonal expansion of the tumor sample.  This observation suggests that integration may lead to cellular proliferation and cancer formation~\cite{Moore2010}.

Studies of the impact of viral integration on tumorigenesis have found evidence of recurrent genomic hotspots in both HBV-related hepatocellular carcinoma (HCC; ~\cite{Ding2012, Khoury2013}) and HPV-related cervical cancers~\cite{Schmitz2012}.  In these studies recurrent integrations were often found in or near known oncogenes such as TERT, MYC, and MLL4, suggesting that integration occurs randomly across the genomic, however over the course of tumor formation progression, integrations in key genomic regions may provide a selective advantage for the host cells, resulting in recurrent integration sites across different samples.  

The majority of HPV and HCC tumors, however, do not contain recurrent integration sites.  Thus, focusing on just recurrent integration sites may provide an incomplete story of the role of viral integration on tumor formation.  Integration can result in altered mRNA expression of flanking regions and viral-human fusion mRNA transcripts~\cite{Tang2013}.  Viral-human fusion transcripts can alter functional pathways, as in the case of the viral-human fusion transcript HBx-LINE1~\cite{Lau2014}, which acts as a sponge for miRNA-122 and promoted hepatic cell epithelial-mesenchymal transition (EMT)-like changes and increased susceptibility to induced tumor formation~\cite{Liang2016}.  

Historically, viral integration was detected using PCR-based methods, however due to the labor intensive process of running the PCR experiments, and the low coverage and depth of PCR techniques, integration events were difficult to identify.  With the advent of Next Generation Sequencing (NGS) technology, human genomes can now be sequenced with high coverage and high sequencing depth thus making it more feasible to accurately identify the location of viral integrations within the human genome.  Many pipelines have been developed recently for the detection of viral integration from paired-end Illumina data (VirusSeq~\cite{Chen2013}, VirusFinder~\cite{Wang2013}, ViralFusionSeq~\cite{Li2013} in 2013; VERSE~\cite{Wang2015}, Virus-Clip~\cite{Ho2015}, and Vy-PER~\cite{Forster2015} in 2015).  While each pipeline varies in how detection is performed, the overarching theme is similar: identify single end or paired-end reads that map to both the human and viral reference genome.  Paired-end reads in which one read maps to a viral reference and the other read maps to a human reference (known as \emph{chimeric paired-end reads}) are indicative of a potential integration event.  Single end reads that partially map to a human reference and viral reference are known as \emph{split end reads} and are useful for determining the exact integration point in the human chromosome.  The key step for these methods is the use of a reference-based read mapper (i.e., BLAST~\cite{Altschul1990}, BLAT~\cite{Kent2002}, BowTie2~\cite{Langmead2012}, or BWA~\cite{Li2009}) to categorize the read pairs.  These methods perform pairwise sequence alignment between the query sequence and a reference sequence in order to determine if a) the query sequence is related to the reference, and b) the optimal alignment of the query sequence to the reference sequence.

Detecting viral integrations from NGS technology is still challenging.  For example, the integrations from ~\cite{Hu2015} have been challenged as false positives integrations as a result of PCR sequencing artifacts~\cite{Dyer2015}.  Indeed, one of the large problems with viral integration detection lies in separating out true integrations from false positives integrations, which can result from shared repeat regions between human and viral genomes~\cite{Forster2015}.  

One challenge of reference-based pairwise alignment is in aligning reads from novel or evolutionarily divergent organisms~\cite{Brenner1998,Park1998}.  This can result in a poor alignment between the query sequence and the closest reference sequence or can even result in the failure to detect any evolutionary relationship between the query sequence and the known reference sequences.  An alternative approach is to first represent the relationships between the reference sequences using an evolutionary model.  Then a given query sequence can be scored against the model, and if the score is significant, fitted to the model.  One example of a commonly used evolutionary model is a \emph{profile Hidden Markov Model} (HMM;~\cite{Eddy1998}). HMMs are a statistical model for representing a multiple sequence alignment (MSA), and HMMs have been used to detect membership of query sequences to protein families~\cite{Finn2010}, align fragmentary sequences~\cite{Eddy1998}, and identify viral fragments from metagenomic samples~\cite{Skewes-Cox2014}.  Given a query sequence, the profile HMM can produce scores that represent the probability of the model generating the sequence, as well as the best alignment of the query sequence to the model.

Previous research has shown that profile HMMs can have difficulty in recognizing sequences that come from evolutionarily divergent families, and that using an ensemble of profile HMMs (eHMMs) can improve recognition rates~\cite{Mirarab2012,Nguyen2014,Nguyen2015,Nguyen2016_hippi}.  The eHMMs technique has been shown to have better detection and alignment of query sequences to a given model compared to the single HMM approach, and as a result, better downstream downstream phylogenetic analyses. 

Here, we present \textbf{V}iral \textbf{I}ntegration and \textbf{F}usion \textbf{I}dentification (ViFi), a new tool for detecting viral integrations from WGS data and human-virus fusion mRNA from RNASeq data (Fig.~\ref{flowchart}).  Unlike other viral integration detection pipelines that use reference-based alignment mapping for identifying viral reads, ViFi uses a combination of reference-based alignment mapping and eHMMs to identify viral reads.  In addition, ViFi incorporates the mappability scores to reduce false positive detections.  The end result is a tool that can accurately detect viral integrations with high precision and recall, even when the viruses are highly mutated or are not found in the reference Papillomavirus genomes.  

We show a comparison ViFi to competing tools, VERSE and ViralFusionSeq, on both simulated and biological datasets with experimentally verified integrations.  These datasets include simulated WGS of genomes containing integrated HPV (HPV-SIM), WGS from HCC samples taken from patients with HBV~\cite{Sung2012} (HCC-WGS), RNA-seq datasets from HCC cell lines infected with HBV~\cite{Lau2014} (HCC-RNA).  We also compared the fusion mRNA sequences found by ViFi and VERSE to those from another study on fusion human-viral mRNA transcripts~\cite{Tang2013} on tumor cervical samples taken from the The Cancer Genome Atlas (TCGA-CESC).  Finally, we perform a comprehensive analysis on the impact of HPV integration on transcription expression using matched paired WGS and RNA-seq data from the TCGA-CESC datasets.  Details for each of the datasets is provided in the \textbf{Datasets} section, and a summary is provided in Table~\ref{table:data}. 

%We compare ViFi with VERSE~\cite{Wang2015} and ViralFusionSeq~\cite{Li2013} on both simulated and real biological datasets.  We selected VERSE and ViralFusionSeq for comparison as VERSE is among the most recent viral integration tools and has been shown to outperform VirusFinder and VirusSeq, and ViralFusionSeq is the only viral integration tool available on NIH HPC system.  We report the true positive and false positive integration points reported on the simulated datasets, and the true positive integration points reported on biological datasets with experimentally verified integration points.  Finally, we report the number of fusion transcripts detected from the matched RNA-seq datasets, and the number of those transcripts that are supported from the matched WGS datasets.

\section*{Results}

\paragraph{\textbf{Comparison on simulated datasets.}}  We simulated WGS datasets with integrated viruses, exploring the impact of viral strain diversity and number of integrations on detection accuracy and computational running time.  The integrated viruses included three HPV16 strains simulated with either low, medium, or high rates of substitution mutation, and one papillomavirus (AgPV1) that is not included the reference PV database to simulate detection of a novel virus.  The number of integrations varied from 25, 100, 500, and 1000.  For each model condition (parameterized by the integrated virus and number of integrations), we generated five replicate WGS datasets at 25x coverage (approximately 50 GB of data per dataset) for a total of 16 model conditions and 80 datasets.  

We ran ViFi, either with using only the reference-based mapping or using the combination of reference-based mapping and eHMMs, VERSE, and ViralFusionSeq on the simulated datasets (Fig.~\ref{sim_results}.  All methods were run on a dedicated compute node and given 48 hours to complete.  Only ViFi was able to complete within the allotted time on all datasets. VERSE completed only on datasets with less than \textbf{XXX [NOTE:  Need to wait for runs with 100, 500 integrations to complete]} integrations, while ViralFusionSeq failed to complete on any of the datasets.  Due to the computational cost of running ViralFusionSeq, we excluded it from the remainder of this study.

Both ViFi (both with and without HMMs) and VERSE have high precision and recall on the easier datasets (95-99\% similarity with HPV16).  However, when the integrated virus has only 90\% similarity to HPV16, VERSE's precision and recall drops by one half.  ViFi, on the other hand, is still able to accurately detect the highly mutated simulated HPV16 sequence.  On the dataset that contains the novel AgPV1 sequence, only ViFi with the HMMs is able to accurately detect integration events (88\% recall and 93\% precision).  ViFi without the HMMs is only able to detect 20\% of the integrations, but with high precision.  VERSE is unable to detect any integration events.  

Interestingly, VERSE takes significantly longer when it detects more integrations.  When no integrations are detected, it takes roughly 6 hours to complete, however, this time jumps by two to three hours when it detects 20 or more integrations.  Thus, VERSE fails to complete on any dataset with more than \textbf{XXX} integrations.  ViFi, on the other hand, had an average increase of running time of 32 minutes when increasing the number of simulated viral integrations from 25 to 1000 integrations, without any loss of recall or precision.  As some cervical cancer samples can have up to 100 to 600 HPV integration events~\cite{Hu2015}, it is vital that integration pipelines are able to run efficiently on samples with many integrations.\textbf{NOTE:  Simulations being run for 100 and 500 integrations, will be able to better discuss scaling once complete.}


%We ran ViFi, either with using only the reference-based mapping or using the combination of reference-based mapping and eHMMs, VERSE, and ViralFusionSeq on the simulated datasets that contained 25 viral integrations per dataset (Fig.~\ref{sim_results}).  %All methods were run on a dedicated compute node with 24 CPUs and given 48 hours to complete.  Only ViFi and VERSE completed within the allotted time; ViralFusionSeq failed to complete on any of the datasets.  ViFi required an average wall clock running time of 7 hours and 30 minutes (minimum of hours 13 minutes to maximum of 8 hours and 2 minutes).  The impact of using HMMs had minimal impact on the running time of ViFi (average increase of 14 seconds).  VERSE required an average of 8 hours and 19 minutes to complete per dataset (minimum of 6 hours and 1 minute to a maximum of 9 hours and 19 minutes).  Due to ViralFusionSeq being unable to complete on these datasets, we exclude ViralFusionSeq from the remainder of our comparison study.

\begin{figure}[htpb]
  \centering
  \subfloat[Precision-recall on simulated datasets.]{\includegraphics[width=4in,frame]{{results/simulation.main}.pdf}}\\
  \subfloat[Running time on simulated datasets.]{\includegraphics[width= 4in,frame]{{results/running}.pdf}}  
\caption[Results on simulated datasets.]
{\label{sim_results}  {\bf Results on simulated datasets.}  We report a) the recall and precision for four different model conditions with 25 simulated viral integrations, and b) wall clock running time for the easy model HPV16 model condition.  We show results for when we include the usage of the  ensemble of HMMs in detecting viral reads (Default) and when we exclude the usage of the ensemble of HMMs (Default-NoHMMs).  The first three model conditions (easy, medium, and hard) vary the percent similarity of simulated HPV16 genomes to the reference HPV16 genome, with five replicates per simulation.  The last model condition uses Alouatta guariba papillomavirus 1 (AgPV1), a PV genome not included in the set of viral genomes to simulate detection of a novel HPV virus.  AgPV1 is 44\% similar to HPV16.  VERSE failed to detect any viruses on the AgPV1 model condition.  All datasets were simulated with 25x coverage.}
\end{figure}


\paragraph{\textbf{Comparison on biological datasets with experimentally verified genomic integrations.}}  Next, we compared the methods on HCC-WGS and HCC-RNAseq datasets with experimentally verified HBV integrations.  Of the 21 experimentally verified integration points in the HCC-WGS dataset, ViFi was able to detect 17 of the integration points, and VERSE was able to detect 16.  In the HCC-RNAseq datasets, both ViFi and VERSE recovered 10 out of 11 verified fusion mRNA points (Fig.~\ref{bio_results}).

\paragraph{\textbf{Comparison of fusion mRNA detection on TCGA-CESC dataset}}
A 2013 study of RNA-seq datasets from the TCGA database by Tang et al.~\cite{Tang2013} explored the landscape of human-viral fusion gene expression.  The authors found that fusion mRNA transcripts are often found in HPV-related and HBV-related cancers.  The authors verified the fusion transcripts found in a small subset of the datasets (8 out of the 178 datasets) by showing that the fusion transcripts were concordant genomic integration events found with the matched WGS data.  We expand upon this study by comparing the concordance of fusion mRNA transcripts and genomic integration events on 28 TCGA cervical cancer samples (TCGA-CESC) with matched WGS and RNA-seq data.

For each sample, we ran ViFi and VERSE~\textbf{NOTE:Currently running on XSEDE, update this section once done} on the WGS data to detect genomic viral integrations.  For each viral integration reported by ViFi and VERSE, we report whether or not each of the two methods and the Tang et al. study also found one or more fusion mRNA events within a 100kb interval around that integration point.  A Venn diagram showing the overlap of fusion mRNA events found by the different methods show that for any case in which VERSE or Tang et al. found a fusion mRNA event, ViFi also detected the event (Fig.~\ref{venn_diagram}).  In addition, ViFi detected five fusion mRNA events that were also supported by the WGS data that neither VERSE or Tang et al. identified.  Thus, not only use ViFi detect more mRNA fusion transcripts, those transcripts are more likely to represent true fusion transcripts as the fusions are in concordance with genomic integration events. %Closer inspection of these events show strong evidence that the fusion mRNA sequences represent true fusion events (~\textbf{Discuss best way to show this, can see visually by eye using IGV that data from WGS/RNA-seq support this}).

\begin{figure}[htpb]
  \centering
  \includegraphics[width=1\linewidth]{results/tcga_triple.pdf}\\
\caption[Venn diagram of fusion reads recovered from the RNAseq data from Larsson 2013 study]
{\label{venn_diagram}  {\bf Venn diagram of fusion read integration points recovered from the TCGA-CESC samples with matched RNA-Seq and WGS sequencing.}  We show a Venn diagram of the overlap of the WGS integration points with a matching mRNA event within 100kb reported by ViFi, VERSE, and the Tang et al. 2013 study on the TCGA-CESC samples with both RNA-Seq and WGS sequencing matched pair data.  }
\end{figure}

\paragraph{\textbf{Genome-wide analysis of genomic integrations and fusion mRNA sequences in cervical cancer samples.}}
The previous analysis examined the concordance of fusion mRNA and genomic integrations on 28 TCGA-CESC samples explored in the Tang et al. study.  We expanded upon this analysis by exploring the landscape of genomic integration and fusion events across all TCGA-CESC samples with matched WGS and RNA-seq data.  In addition, we also examined the impact of integration on transcription expression.  

We detected integrations in 51 samples of 68 samples (226 integrations total).  For each integration, we examined the number of annotated genomic features within 10kb of the genomic integration(Fig.~\ref{integration_counts}) and compared the distribution of observed annotations to the expected annotations resulting from random intervals.  In particular, we focus on whether there is an enrichment of Long terminal repeats (LTR) and Long interspersed nuclear element (LINE) annotations near genomic integrations as these elements are typically not expressed in normal samples~\cite{Kinomoto2007,Sigurdsson2012}, but can be transcriptionally active in tumor samples~\cite{Romanish2010,Rodic2013,Xiao-Jie2016}.

Previous research has observed that genomic regions with HPV integrations were enriched for genes~\cite{Akagi2014} or long terminal repeat/endogenous retroviruses (LTR/ERV1)~\cite{Hu2015} elements.  We also observed a significant enrichment of genes near integration points (Z-test; p-value $<$ 0.004).  We also detected a depletion of all LTR elements near integration points (Z-test; p-value $<$ 0.013), though after multiple hypothesis correction, this observation was not significant.  Finally, we detected a slight enrichment of oncogenes and no enrichment or depletion of LINE elements.  While many studies have noted that integrations do occur at or near oncogenes, these are very rare, and thus, can only explain tumorigenesis in the few cases in which they occur.  


\begin{figure}[htpb]
  \centering
  \includegraphics[width=1\linewidth]{{results/expected.counts.violin}.pdf}\\
\caption[Annotations of integrated regions.]
{\label{integration_counts}  {\bf Observed and expected number of annotations of the integrated regions.}  We show the violin plots of the expected number of counts from random integrations for each annotation over 1000 simulated experiments, and the actual number of annotations for the observed integrations found in our TCGA-CESC dataset (in red).  The quantiles of the violin plots are displayed with lines.  The p-values of the observed annotation counts (Z-test) are: LINE $<$ 0.882;	LTR $<$ 0.013;  Gene $<$ 0.004; and	Oncogenes $<$ 0.179.}
\end{figure}

We observed recurrent integrations (within 50kb of each other) in two loci: two samples with integrations between region chr13:73955151-74005092 and four samples with integrations between region chr8:128747810-128889296.  The first region contains no nearby genes.  The latter region contains the MYC and PVT1 genes and had been found to contain recurrent integrations from previous studies~\cite{Hu2015}.  However, the vast majority of integrations are non-recurrent and occur uniquely within that particular sample, and thus, focusing on integrations that occur in genomic hotspots found across samples will exclude the majority of integrations and their roles on tumorigenesis.

Next, we examined the likelihood of genomic integrations resulting in fusion transcripts.  We found that 52\% of the genomic integrations (119 genomic integrations) had at least one fusion mRNA sequences.  We detected a total of 376 fusion mRNA sequences, of which 88\% could be assigned to a genomic integration; the remaining 12\% were likely to be either false positive fusion detection or a false negative genomic integration detection.  70\% of the fusion transcripts could be mapped to genomic regions within 10kb of a genomic integration (Fig.~\ref{distance_breakpoint}), and 92\% of the fusion transcripts mapped to regions within 50kb of a genomic integration.  As previous studies have shown that HPV transcripts and HPV fusion transcripts undergo alternative splicing~\cite{Ziegert2003,Johansson2013}, it is not surprising that the human portion of the fusion transcripts might map a large distance away from the genomic integration point.  These results show that ViFi is able to recover fusion events that are concordant with the genomic data in an automated fashion.

\begin{figure}[htpb]
  \centering
  \includegraphics[width=1\linewidth]{results/{wgs.rna.histogram.distances}.pdf}\\
\caption[Distance of fusion mRNA breakpoint to nearest WGS integration breakpoint.]
{\label{distance_breakpoint}  {\bf Histogram of distance of fusion mRNA breakpoint to nearest WGS integration breakpoint.}  }
\end{figure}

As the RNA-sequencing library preparation was not strand specific, we cannot determine which strand (for either the human or viral portion) the fusion transcripts originated from.  However, because HPV genes are known to be transcriptionally active in cervical cancer cells, we can assume that the majority of viral transcripts should be orientated in the same direction as the viral gene.  Using this assumption, we can infer whether the human portion of a fusion transcript is upstream or downstream of the viral gene.  For each fusion mRNA sequence, we examined the proportion of chimeric paired-end reads that support whether human portion is upstream or downstream of the viral gene (Fig.~\ref{mrna_directions}).  The 82\% of fusion mRNA sequences showed that the human portion was downstream of the viral genes.  Of the 18\% fusion mRNA sequences that showed that the human portion was upstream of the viral gene, 87\% were within 10kb of an annotated gene.  These observations suggest that transcription of fusion mRNA sequences are largely being driven by the upstream regulatory elements within the viral genome.  In the rare cases in which the viral gene is downstream of the human portion of the fusion transcript, we suspect that regulatory elements in the human genome are driving this expression.  

\begin{figure}[htpb]
  \centering
  \includegraphics[width=1\linewidth]{{results/rna_direction.summary.histogram}.pdf}\\
\caption[Fusion reads supp.]
{\label{mrna_directions}  {\bf Orientation of the mRNA fusion transcripts.}  We report whether the human portion of the fusion mRNA sequences are upstream or downstream of the viral portion.  The orientation is inferred from the paired-end reads under the assumption that the viral read is always oriented in the direction of the viral gene, and support for the orientation is computed as the proportion of paired-end reads supporting that orientation.  In addition, we report whether there is a gene within 10kb of the fusion mRNA read.}
\end{figure}

The previous result exploring the orientation of the fusion mRNA sequences showed a strong bias of human portions almost always being downstream of the viral genes.  This observation suggested that viral integration might strongly impact expression near the integration point.  In order to confirm this hypothesis, for each integration, we examined the transcription activity of the region within 10kb of the sample containing the integration and compared the expression against the mean transcriptional activity of the same region for samples without that integration.  This gives us a baseline to compare expression change caused by integration within a genomic region.  We then computed the log-fold change in expression as an estimate of the impact of integration on expression (Fig.~\ref{all_expression}).  The results show that integration typically results in either a neutral or increase in log-fold change in expression.  This change is more prominent when the integration also results in a fusion transcript, and can be as large as a 262 fold increase in expression compared to the same region in samples without integrations.  Thus, a single integration might result in a large disruption of regulatory network, depending on what regulatory elements are nearby.  

\begin{figure}[htpb]
  \centering
  \includegraphics[ width=1\linewidth]{{results/wgs.fpkm.fusion.boxplot.10000}.pdf}\\
\caption[Log-fold change of expression between genomic segments with and without integrations.]
{\label{all_expression}  {\bf Boxplots of log-fold change of expression between genomic segments with and without integrations.}  We show boxplots of the Log2-fold change in expression of genomic regions with and without integrations.  The genomic regions are separated by whether or not there are associated fusion transcriptions with the genomic integration.}
\end{figure}

When we average the log-fold expression change across all genomic segments containing integrations, we find that there is an increased expression of nearby human mRNA (2.83 fold increase; Fig.~\ref{expression_transcripts} and Supplemental File 1), and moreover this change in expression is impacted on whether the integration resulted in a fusion transcript.  There is a 7.41 fold increase if the genomic segment contains a fusion mRNA sequence, and a 0.97 decrease when there is no fusion mRNA present.  More surprisingly, this change in expression affects not only genes, but all elements within the genomic interval.  When we examined regions that had known annotations (LINEs, LTRs, genes, or oncogenes), we found that that expression was significantly enhanced in the annotated regions, with a 1.70 to 2.43 fold increase in expression for LINEs, LTRs, and genes, with an increase to 2.55 to 5.53 fold when fusion mRNA is present.  These results show that integration can not only increase transcriptional activity, but can also result in transcript of elements that are normally silent in normal cells, such as LINEs and LTRs.  As LINE-1 elements (L1) have been shown to be upregulated in some cancers~\cite{Rodic2013,Xiao-Jie2016}, this result highlights that all integrations, not just recurrent integrations, can play a powerful role on local transcriptional activity.  

%, but a fold change of 0.86 to 0.92 when no fusion mRNA is presents


\begin{figure}[htpb]
  \centering
  \includegraphics[ width=1\linewidth]{{results/expression.summary.barplot.fold}.pdf}\\
\caption[Expression of mRNA transcripts in genomic segments with and without integrations.]
{\label{expression_transcripts}  {\bf Log2 fold change in expression of mRNA transcripts in genomic segments with and without integrations.}  We report the Log2 fold change in expression of annotated features between genomic segments with and without integrations.  We report the average fold change for all segments, segments that contain fusion mRNA sequences, and segments without fusion mRNA sequences.  For a given integration in a sample, we select a 10kb flanking region around the integration site.  We then report the logfold change in expression level in samples that have an integration in this region and samples that do not.}
\end{figure}

To better characterize the impact of integration on expression, we examined the transcriptional activity local to the integration site.  For each genomic integration, we categorize the integration as a simple, complex, or fusionless integration site (Fig.~\ref{updown}a).  We define simple integrations as a single integration within the 10kb interval with the chimeric paired-end reads being concordant, allowing us to determine which direction is upstream and downstream of the integration site.  We define complex integrations as either having two or more integrations within the 10kb interval that are sufficiently far apart that we can detect them as being separate integrations, or a cluster of integrations that is detected as a single integration event.  This cluster can produce multiple fusion mRNA sequences and result in a set of discordant chimeric paired reads.  Both cases of complex integrations make it difficult to determine what is upstream or downstream of the integration site.  Finally, we define an integration as a fusionless integrations if it has no chimeric mRNA paired-end reads that can be used to determine orientation.

The first observation is when there is a fusion mRNA present in the interval (simple or complex), there is a two- to five-fold increase in expression local to the integration point (Fig.~\ref{updown}b).  The increase in expression can be evident up to 100kb around the integration point (Fig.~\ref{updown_250kb}).  In the case of simple integrations, there is a clear increase downstream of the integration point.  Finally, integrations without any fusion mRNA sequences show slight decrease in expression.  These observations show that integration, in general, increases local transcriptional activity near the integration site, with a strong bias downstream of the integration point for simple integrations.  This driving factor is agnostic to whatever is downstream of the integration, and thus, can result in transcription of elements that normally silent or not expressed in normal samples. 

\begin{figure}[htpb]
  \centering
  \subfloat[Classification of integration points.]{\includegraphics[width=4in,frame]{{classification}.pdf}}\\
  \subfloat[Logfold expression near integration point.]{\includegraphics[width= 4in,frame]{{results/updown.10000}.pdf}}
\caption[Fusion reads supp.]
{\label{updown}  {\bf Categorization of integration points and expression change around integration point}.  a) We categorize integration points into three categories: simple integration, complex integration, and no fusion mRNA integration.  Simple integrations have no other integrations within a 10kb window, and at least 75\% of the chimeric paired-end reads supporting a fusion mRNA event are orientated in the same direction.  Complex integrations either have multiple integration events within a 10kb window, have multiple fusion mRNA supporting different orientations, or have chimeric paired-end reads that support conflicting orientations for the same mRNA fusion event.  Integrations that have no fusion mRNA events are categorized in a separate category.  b) We report the average log-fold expression change upstream and downstream of an integration event.  The position is reported relative to the integration point in the human genome, with negative position being upstream of the integration event, and positive position being downstream of the integration event.  The results are separated out by the integration category.  In total, there are 68 simple integrations, 51 complex integrations, and 107 integration events with no fusion mRNA sequences.} %35, 110, 81
\end{figure}

\section*{Discussion}
We present ViFi, a new method for detection of viral integration sites and fusion mRNA sequences.  The key advances of ViFi are the use of eHMMs to better detect evolutionarily divergent viruses and the use of mappability scores to reduce false positive detections.  Our simulation experiments shows that ViFi has high precision and sensitivity in detecting viruses, even under difficult conditions of highly mutated or novel viral strains.  VERSE, on the other hand, performs poorly or fails to detect integration on the hardest model conditions.  On the biological datasets, ViFi detects more integration events than VERSE.  ViFi is computationally efficient and its running time is not dramatically impacted the number of viral integrations, whereas VERSE is unable to complete analyses within 48 hours on datasets with 1,000 integrations.

Previous research on the role of viral integration on tumorigenesis focused on identifying genomic hotspots containing recurrent integrations and identifying change in both host and viral gene expression resulting from integration~\cite{Tang2013,Lawrence2015,Hu2015,Zhang2016}.  Previous observations showed that recurrent integrations often occurred near known oncogenes such as MYC, PVT1, ERBB2, RAD51~\cite{Tang2013}, but our results show that recurrent integrations are very rare.  Hu et al. 2015 suggested that HPV integrates randomly across the genome until an integration occurs in a region that provides a proliferative advantage, resulting in oncogenesis.  However, the vast majority of integrations are non-recurrent and the majority of samples do not have integrations in or near oncogenes.  Thus, focusing on genes with recurrent integrations can only explain samples that have integrations in those genes, potentially ignoring a large portion of samples that do not contain integrations in recurrent regions.

Our results show that viral integration plays a powerful role on local transcriptional activity, especially when fusion human-viral mRNA sequences are present.  This includes a strong increase in transcription near the integration site, as well as transcription of elements that might normally not be expressed.  We hypothesize that this increase is a result of recruitment of transcription factors by HPV's upstream regulatory region (URR) and subsequent transcription read-through to produce both viral-human mRNA transcripts as well as an uptick of expression downstream from the viral integration point.  When there are no fusion mRNA transcripts present, it is possible that the URR has been disrupted after integration and is unable to recruit transcript factors.  Thus, we do not observe the same increase in expression for integrations without fusion mRNA sequences.

One key question is why some integrations result in fusion mRNA transcripts and others do not, and why this difference causes a large discrepancy between expression levels for integrations with and without fusion transcripts.  One possibility is that the fusion transcripts we detect may be originating from episomal structures that are not bound to the chromatin and thus are easily accessible.  HPV can exist in both integrated and episomal forms within cervical cells~\cite{Shin2014,Williams2011}, and cells from cervical lesions caused by HPV can have two to five times as many micronuclei compared to normal cervical cells~\cite{Cortes-Gutierrez2010}.  Thus, it would not be implausible for there to exist a micronucleus that contains a viral segment or a viral episome that contains a human segment.  Such structures could be represented as a cyclic genome, and indeed, we have found several examples in which we can arrange human segments and viral segments into an episomal structure.  For example, chimeric reads from the sample TCGA-C5-A0TN suggest that a segment from chromosomes 2 and 3 are connected, a segment from HPV genome and chromosome 2 are connected, and a segment from chromosome 3 and the HPV genome are connected  (Fig.~\ref{tcga_c5_a0tn}).  Such a structure could be indicative of an episome, which would be unbound to chromatin and thus able to transcribe freely.  Perhaps some integrations with fusion mRNA sequences might be a result of human-viral episomal structures.  This is an exciting hypothesis as such structures could replicate independently of the cell, and episomes which provide reproductive advantage might be selected for amplification.  Such an idea might explain why there is flanking local copy number amplification often found near integrated sites~\ref{Akagi2014}.  Future fluorescence in situ hybridization (FISH) may help determine the structure of these integrations.

\begin{figure}[htpb]
  \centering
  \includegraphics[width=0.6\linewidth]{{results/episome}.pdf}\\
\caption[Potential structure of TCGA-C5-A0TN.]
{\label{tcga_c5_a0tn}  {\bf Hypothesized structure for TCGA sample TCGA-C5-A0TN.}  Chimeric reads supports a portion of chromosomes 2 and 3 and the HPV genome being connected.}
\end{figure}

Our results suggests another potential mechanism for viral-related oncogenesis: genomic instability and pathway disruption via expression of LINE and LTR elements that are not normally expression via transcription read-through.  L1 elements are retrotransposons that have a ``cut and paste'' mechanism which allow them to replicate through RNA intermediates and re-insert back into genomic DNA.  L1 elements have been shown to create double stranded breaks in the host cell genome, which might lead to genomic instability~\cite{Gasior2006}.  L1 elements are typically epigenetically repressed in somatic cells~\cite{Kinomoto2007,Sigurdsson2012}, but they have been shown to be active in selected neoplasms~\cite{Rodic2013,Xiao-Jie2016}.  Interestingly, the \emph{p53} gene will often activate the apoptotic pathway when an L1 re-integrates into the genome, however, this can be blocked by the HPV E6 protein~\cite{Haoudi2004}.  LTRs can act as promoters, enhancers, and transcriptional factor-binding sites, and can potentially be transcriptionally active, depending on the tissue~\cite{Yu2013}.  In most cases, LTRs are transcriptionally silenced by epigenetic mechanisms, however, in cancer, LTRs have been shown to have up-regulated expression (see~\cite{Romanish2010} for a review).  Our results show more than an order of magnitude fold increase in expression of these elements in regions proximal to integration sites.  As HPV randomly integrates into the genome, it might fall proximal to LINE and LTR elements and result in deregulation of cellular pathways by the active transcription of LINE and LTR elements, and in particular increased genomic instability via production of L1 elements.  

The production of fusion transcripts may also play a role in tumor progression.  As previously noted, HBx-LINE1 has been shown to result in increased susceptibility to liver cancer by acting as a sponge for miRNAs~\cite{Lau2014,Liang2016}.  Recent studies for HPV-related cancers have noted differential expression profiles of miRNAs for HPV-positive and HPV-negative samples~\cite{Lajer2012,Gao2016}.  One possibility is that fusion transcripts produced by integrated HPV might also act as a sponge for miRNA.  

The impact of viral integration on proximal transcription expression is clear.  Our study highlights the need for more research on the role of HPV integration on oncogenesis, and in particular, with more focus on samples with non-hotspot integrations.  As the majority of integrations are not occurring in genomic hotspots, mechanisms such as increased LINE and LTR expressions might play a crucial role in tumor progression and genomic instability.  

%One possible direction to better understand the role of LINE and LTR expression is combining proteomics and genomics.  In particular, focusing on whether HPV positive samples with integration show higher expression of L1 encoded proteins, ORF1p and ORF2p, compared to HPV positive samples without integration.  
 
\section*{Method}
\paragraph{\textbf{ViFi.}}
We present \textbf{V}irus \textbf{I}ntegration and \textbf{F}usion \textbf{I}dentification (ViFi), a new pipeline for detection of integration sites (Fig.~\ref{flowchart}).  Unlike other integration detection pipelines that use reference-based alignment mapping for identifying viral reads, ViFi uses a combination of reference-based alignment mapping and ensemble of profile HMMs to identify viral reads.  

\paragraph{\emph{Pre-processing.}} ViFi begins with a pre-processing step (see Fig.~\ref{flowchart}a) that builds a BWA index for the human reference genome and viral genomes (Hg19+viral index).  In addition, ViFi takes the genomes from the viral family and estimates a MSA on each family using PASTA~\cite{Mirarab2014}.  A maximum likelihood (ML) tree is estimated from the alignment using RAxML~\cite{Stamatakis2014}.  Next, ViFi builds an ensemble of HMMs from the alignment and tree using HIPPI~\cite{Nguyen2016_hippi}.  Briefly, a profile HMM is computed on the input MSA and ML tree.  Next, the ML tree is partitioned into two subtrees by taking its centroid edge (i.e., the edge that best separates the tree into two subtrees with roughly equal number of leaves).  Profile HMMs are then built on the alignments induced by the leaf set of each of the subtrees.  The process repeats recursively on each subtree until there are at most 10 sequences in the subtree.  This results in a collection of nested hierarchical profile HMMs which we call the ensemble of HMMs.  This pre-processing step of building the ensemble of HMMs only needs to be run once for each viral family of interest.

\paragraph{\emph{Identification of candidate reads.}} Similar to other existing viral integration detection pipelines, the first step of ViFi is to align each paired-end read to the Hg19+viral index (see Fig.~\ref{flowchart}b).  The paired-end reads are separated into four different groups: i) reads in which both paired-end reads mapped Hg19 or both to the viral genomes, ii) paired-end reads in which one end mapped to Hg19 and the other to a viral genome, iii) paired-end reads in which one end mapped to Hg19 and the other is unmapped, and iv) all other reads.  Typically, most existing viral integration detection pipelines focus on the set of reads in group ii) and discard reads found in group iii).  However, a read that is viral in origin might be unmapped because it is too evolutionarily divergent from the set of known viral genomes.  Our method attempts to rescue paired-end reads in this category by scoring the unmapped reads against the ensemble of HMMs created in the pre-processing step.  If a read has a sufficient score to one of the HMMs in the ensemble (E-value of 0.01 or lower), then the read is marked as a viral read and the paired-end read is put into the candidate set, along with all reads from group ii).  This step allows the detection of novel or evolutionarily divergent viral sequences belonging to the same family.  The last step is to infer integration from the set of candidate reads.  

\paragraph{\emph{Identification of integration point.}}  The set of candidate reads are ordered by coordinates of the human genomic position.  Read pairs with poor mappability scores, defined as Duke Uniqueness Score~\cite{unknown} less than 0.33 or MAPQ score less than 10 are removed.  These removed reads represent reads that might map to multiple locations.  The remaining read pairs within 300 bps of each other are clustered into a group of read pairs supporting the integration point.  The reads remaining in the cluster are then considered highly supported reads (i.e., have high mapping score).  

%Any paired end reads in which one or more ends did not map to the reference are put into a candidate set.  From the candidate set, all reads are scored against the ensemble of HMMs, and those that score above a threshold are marked as viral.  paired end reads that have at least one read mapping to the human reference and one marked viral are considered candidate integration points.  Integration points with at least three paired end reads supporting the integration are listed as positive integrations.


%paired end reads in which at least one end did not map to the human reference are put into a candidate set.  All reads in the candidate set are scored against each HMM in the ensemble using nhmmer from the HMMER suite of tools~\cite{Eddy1998}.  Any read that scores above a threshold $T$ is marked as viral.  paired end reads in which at least one end maps to the human reference and at least one end is marked as viral are considered as candidate integration event.  Any read that maps to the human reference and is marked as viral is candidate split read and is used to identify the integration point on the human genome.  If an integration point is covered by a split read, we can identify the exact integration location on the human genome.  However, as not all integration events are covered by split reads, we also identity integration points with a range (ADD MORE DETAIL) defined by the insert size.  Integration points that are supported by three or more paired end reads are outputted as  true integration points.

\begin{figure*}[htpb]
  \centering
  \subfloat[Pre-processing]{\includegraphics[width=4in,frame]{{preprocess}.pdf}}\\
  \subfloat[Pipeline]{\includegraphics[width= 4in,frame]{{mapping}.pdf}}
\caption[Overview of integration detection pipeline.]
{\label{flowchart}  {\bf Overview of integration detection pipeline.}  Integration detection is split into two phases.  In the a) pre-processing step, a BWA index is created from the human reference genome and input viral genomes (Hg19+viral).  In addition, a multiple sequence alignment (MSA) is estimated from the viral genomes, and an maximum likelihood (ML) tree is estimated from the alignment.  The MSA is decomposed into an ensemble of profile Hidden Markov models.  In the b) viral detection step, the paired-end reads are mapped against the Hg19+viral index.  Candidate paired-end reads are selected if, i) one end of the read maps to the human genome and the other end maps to a viral genome, or ii) one end of the read maps to the human genome and the other end scores high against the HMM ensemble.  All other reads are discarded.  The integration point is then inferred from the set of candidate reads.
% i) reads in which both paired end reads mapped Hg19 or both to the viral genomes, ii) paired end reads in which one end mapped to Hg19 and the other to the viral genomes, iii) 
%Any paired end reads in which one or more ends did not map to the reference are put into a candidate set.  From the candidate set, all reads are scored against the ensemble of HMMs, and those that score above a threshold are marked as viral.  paired end reads that have at least one read mapping to the human reference and one marked viral are considered candidate integration points.  Integration points with at least three paired end reads supporting the integration are listed as positive integrations.
}
\end{figure*}


\begin{figure}[htpb]
  \centering
  \includegraphics[width=0.6\linewidth]{{ehmm}.pdf}\\
\caption[Ensemble of HMMs technique.]
{\label{ehmm}  {\bf Algorithm for generating the ensemble of HMMs.}  The input is a initial MSA and a maximum likelihood (ML) tree that has been estimated for the MSA. The algorithm begins by adding the HMM built on the MSA to the ensemble. If the MSA has more than 10 sequences, the ML tree is decomposed into two subtrees by deleting the centroid edge (i.e., the edge that produces a maximally balanced split of the sequence set into two sets). The subtrees are used to generate induced alignments. HMMs are built for each induced alignment and added to the ensemble. The process iterates on those subtrees that meet the criterion for decomposition (subset size more than max(10, n/10), where n is the number of sequences in the initial MSA, and mean pairwise sequence identity less than 0.4).}
\end{figure}

% (see Fig.~\ref{viral_discovery}).  The basic idea is to first map all paired end reads to both a human reference genome and to known viral reference genomes.  Second, paired end reads in which both reads map to the human reference are removed.  Third, paired end reads in which both reads map to viral genomes are used to identify potential viral candidates.  Fourth, for read pairs in which one read maps to a viral reference and the other read maps to a human reference (known as \emph{chimeric reads}) are indicative of a potential integration event.  Finally, in some cases, a single read might partially map to a human reference on one end and to the viral reference on another end.  These reads are known as \emph{split-end reads} and are useful for determining the exact integration point in the human chromosome.  The key step for these methods is the use of a reference-based read mapper (i.e., BLAST~\cite{Altschul1990}, BLAT~\cite{Kent2002}, BowTie2~\cite{Langmead2012}, or BWA~\cite{Li2009}) to categorize the read pairs.  These methods perform pairwise sequence alignment between the query sequence and a reference sequence in order to determine if a) the query sequence is related to the reference, and b) what is the optimal alignment of the query sequence to the reference sequence.
%For example, a recent study found recurrent HPV integrations in several oncogenes~\cite{Hu2015}.  The authors hypothesized that HPV might first integrate randomly across the genome, however over the long term, integrations at key loci may provide a selective advantage to the host cells, resulting in hot-spots genes with recurrent integration.  Another HPV study found that more than half of HPV-infected tumors had viral-human fusion transcripts, suggesting that the host-spot locations are transcriptionally active~\cite{Schmitz2012}.  In many cases, integrated HPV genomes have a disrupted E2 gene; the suppression of the viral E2 gene can result in suppression of the tumor suppressor p53 gene in humans~\cite{Akagi2014}. Another recent study found that copy number variations (CNV) were significantly increased near HBV integration locations which the authors suggest that integration may cause chromosomal instability~\cite{Sung2012}.  In addition, the authors found recurrent integration in or near several oncogenes, including TERT, which produces an enzyme that is found in high activity in cancer cells.  Thus, being able to identify integration locations can help us detect hotspot genes with recurrent integrations, identify fusion transcript, find oncoproducts, and examine the chromosomal stability around the integration point.  All of these factors can help us better understand the mechanisms of how a normal cell might transform into a tumor cell upon infection.


%\begin{figure}[htpb]
%  \centering
%  \includegraphics[width=1\linewidth]{{viral_discovery_1}.pdf}\\
%\caption[paired end sequencing viral integration discovery pipeline.]
%{\label{viral_discovery}  {\bf Viral integration and discovery.}  1) The human genome (represented in blue) has a viral integration event (represented in red).  2) The DNA is extracted and fragmented.  Some fragments will consist of only human DNA, some fragments will consist of viral DNA, and some fragments will be chimeric.  3) The fragments are sequenced via Illumina paired end sequencing technology and mapped to onto a human reference and a viral reference.  3) Depending on which reference(s) the paired end reads map onto, pairs are grouped into four categories: a) human, b) viral, c) chimeric, or d) split.}
%\end{figure}

\paragraph{\textbf{Datasets.}}  We use both simulated and biological datasets in our studies.  We describe the datasets below.

\paragraph{\emph{HPV simulated WGS datasets (HPV-Sim)}}
We simulated 16 human genomes with HPV integrations.  In order to examine the impact of viral sequence divergence on integration detection, we simulated different strains of HPV by evolving HPV16 down a phylogenetic tree with differing branch lengths.  We grouped the simulations into three categories, depending on the integrating virus strain's similarity to the reference HPV16 strain: easy (99\% similarity), medium (95\% similarity), and hard (90\% similarity).  In addition, we generated one additional dataset with Brown Howler PV (AgPV1), a papillomavirus genome not included in the set of viral reference genomes to simulate detection of a novel HPV virus.  AgPV1 is 44\% similar to HPV16 and is 65\% similar to the most closely related sequence in the set of reference genomes.


\paragraph{\emph{HCC Sung 2012 WGS dataset (HCC-WGS)}}
88 HCC samples (81 HBV-positive and 7 HBV-negative; both tumor and adjacent tissue) were sequenced by Sung et al.~\cite{Sung2012}.  The authors identified recurrent breakpoints (4 or more samples) in known oncogenes/oncogeneic regions (FN1, TERT, MLL4, CCNE1, ROCK1, SENP5).  A total of 399 breakpoints were identified across the 88 samples.  To confirm recurrent breakpoints, the authors randomly selected 32 breakpoints in six affected genes from 21 samples and validated 22 of these (72\%) via PCR.  Our study used the same 21 samples in our comparison.

\paragraph{\emph{HCC Lau 2014 cell line RNA-seq dataset (HCC-RNAseq)}}
Whole transcriptomes from four HCC cell lines were analyzed using RNA-seq~\cite{Lau2014}.  11 chimeric HBV-human fusion transcripts in three of the cell lines were detected and validated using Sanger sequencing.  Our study included the original six cell line RNA-seq data.  

\paragraph{\emph{TCGA Cervical cancer datasets (TCGA-CESC)}}
We found 68 patient cervical tumor samples in the TCGA database~(\url{https://cancergenome.nih.gov/}) with matched WGS and RNA-seq tumor sequencing data.  Of the 68 samples, 28 samples overlapped with the Tang et al. study~\cite{Tang2013} which examined the landscape of mRNA viral fusion events across the TCGA dataset.  Our study includes a comparison of ViFi and VERSE to the results of the Tang study on the 28 samples, as well as analyses on all 68 samples.


\begin{table*}[htb]
\centering
\caption{\textbf{Overview of datasets}.  We provide an overview of the datasets used throughout this study.  }
\label{table:data}
\begin{tabular}{|l|l|l|r|r|}
\hline
Dataset name & Type & Source & Number of samples & Source \\ \hline
HPV-Sim & WGS &Simulated& 16 & Generated for this study \\ \hline
HCC-WGS & WGS & Biological&20 & ~\cite{Sung2012} \\ \hline
HCC-RNA & RNA-seq & Biological&6 & ~\cite{Lau2014} \\ \hline
TCGA-CESC & WGS and RNA-seq &Biological& 68 & TCGA~\cite{TODO} \\ \hline 
\end{tabular}
\end{table*}

%\subsection{HCC WGS patient results}
%Of the 22 experimentally verified integration points, Viral-HMM was able to detect 21 of the points, VERSE 8 of the points, and ...  Note that in the VERSE paper, the authors reported that they were able to detect 16 out of the 22 points.  We had difficulties in running VERSE on our samples, and tools used within the VERSE pipeline produced very large uncompressed SAM files (up to 1.2 TB in size).  We attempted to fix this by having VERSE output compressed BAM files, which allowed several of the failed runs to complete, but not all.  We suspect that with more efficient use of piping, VERSE could avoid producing many of the large files and would be able to complete on our system and potentially achieve the results reported in their paper.

%\begin{table*}[]
%\centering
%\caption{HCC WGS patient results.  Results on detection of 22 experimentally verified integration points across 13 samples taken from patients with HCC.}
%\label{my-label}
%\begin{tabular}{|l|l|l|l|l|l|}
%\hline
%Sample & Integration breakpoints in human genome & Viral-HMM & Viraj & VERSE                                & VirusSeqFus \\\hline
%145T   & chr19: 30303492                         & Y         &       & Y                                    & Running     \\\hline
%       & chr19: 30303498                         & Y         &       & Y                                    & Running     \\\hline
%177T   & chr3: 196625752                         & Y         &       & N                                    & Running     \\\hline
%180N   & chr2: 216280279                         & Y         &       & Running & Running     \\\hline
%186T   & chr19: 36214005                         & Y         &       & Y & Running     \\\hline
%       & chr19: 36214017                         & Y         &       & Y & Running     \\\hline
%198T   & chr5: 1269387                           & Y         &       & Y                                    & Running     \\\hline
%       & chr5: 1269405                           & Y         &       & Y                                    & Running     \\\hline
%26T    & chr18: 107920                           & Y         &       & Running & Running     \\\hline
%200T   & chr19: 30315003                         & Y         &       & Y & Running     \\\hline
%       & chr19: 30315365                         & Y         &       & Y & Running     \\\hline
%268T   & chr19: 30298787                         & Y         &       & Y                                    & Running     \\\hline
%       & chr5: 1292391                           & Y         &       & N                                    & Running     \\\hline
%43T    & chr3: 196625710                         & Y         &       & N                                    & Running     \\\hline
%46T    & chr5: 1295367                           & Y         &       & N                                    & Running     \\\hline
%70T    & chr19: 36212331                         & Y         &       & Running & Running     \\\hline
%       & chr19: 36212311                         & Y         &       & Running & Running     \\\hline
%71T    & chr3: 196625776                         & Y         &       & N                                    & Running     \\\hline
%       & chr19: 36213141                         & Y         &       & Y                                    & Running     \\\hline
%       & chr19: 36213136                         & Y         &       & Y                                    & Running     \\\hline
%       & chr5: 1292403                           & N         &       & N                                    & Running     \\\hline
%95T    & chr19: 36212564                         & Y         &       & Y                                    & Running    \\\hline
%\end{tabular}
%\end{table*}
\section*{Data Access}

\section*{Acknowledgements}
The results published here are in whole or part based upon data generated by the TCGA Research Network: \href{http://cancergenome.nih.gov/}.


\section*{Disclosure Declaration}
VB is a partner in Digital Proteomics, LLC (DP), which licenses and sells computational tools for analyzing mass spectrometry data. The terms of this arrangement have been reviewed and approved by the University of California, San Diego in accordance with its conflict of interest policies. DP was not involved in the research presented here

%\bibliographystyle{natbib}
%\bibliographystyle{achemnat}
%\bibliographystyle{plainnat}
%\bibliographystyle{abbrv}
%\bibliographystyle{bioinformatics}
%
%\bibliographystyle{plain}
%
%\bibliography{Document}

\bibliographystyle{genres}
\bibliography{main}

\section*{Online supplemental materials}
\setcounter{figure}{0}
\renewcommand{\thesection}{S.\arabic{section}}   
\renewcommand{\thefigure}{S\arabic{figure}}   
\renewcommand{\thetable}{S\arabic{table}}   

\begin{figure}[htpb]
  \centering
  \includegraphics[width=1\linewidth]{results/hcc.pdf}\\
\caption[Recovery of experimentally verified integration sites from biological datasets.]
{\label{bio_results}  {\bf Recovery of experimentally verified integration sites.}  We report the number of experimentally verified integration sites recovered from biological datasets for our method and VERSE.  The datasets include the HCC WGS dataset from the Sung et al. 2012 study (21 experimentally validated integration sites) and the HCC RNAseq cell line dataset from the Lau et al. 2014 study (11 experimentally validated integration sites).}
\end{figure}

%\begin{figure}[htpb]
%  \centering
%  \includegraphics[width=1\linewidth]{{results/simulation.1000}.pdf}\\
%  \caption[Precision-recall on simulated datasets with 1000 viral integrations.] {{\bf Precision-recall on simulated datasets.  \label{sim_results_1000}}  We report the recall and precision for four different model conditions.  We show results for when we include the usage of the  ensemble of HMMs in detecting viral reads (Default) and when we exclude the usage of the ensemble of HMMs (Default-NoHMMs).  The first three model conditions (easy, medium, and hard) vary the percent similarity of simulated HPV16 genomes to the reference HPV16 genome, with five replicates per simulation.  The last model condition uses Alouatta guariba papillomavirus 1 (AgPV1), an HPV genome not included in the set of viral genomes to simulate detection of a novel HPV virus.  AgPV1 is 44\% similar to HPV16.  VERSE failed complete on any datasets.  All datasets were simulated with 25x coverage.}
%\end{figure}

\begin{figure}[htpb]
  \centering
  \includegraphics[width=1\linewidth]{{results/expression.summary.barplot}.pdf}\\
\caption[Expression of mRNA transcripts in genomic segments with and without integrations.]
{\label{expression_transcripts_fpkm}  {\bf Expression of mRNA transcripts in genomic segments with and without integrations.}  We report the mean FPKM expression of annotated features in genomic segments with and without integrations.  For a given integration in a sample, we select a 10kb flanking region around the integration site.  We then report the expression level in samples that have an integration in this region and samples that do not.  In addition, we report the expression level randomly selected segments as a baseline. The p-values for the paired Wilcoxon signed rank test of the difference in FPKM expression for genomic segments with and mean FPKM expression in segments without integrations are: LINE < 1e-12;	LTR < 1e-10;  Gene < 1e-10; and	Oncogenes < NA.}
\end{figure}



%\begin{figure}[htpb]
%  \centering
%  \includegraphics[width=0.75\linewidth]{{results/fusion_TCGA-C5-A0TN}.pdf}\\
%  \caption[Example of fusion mRNA found by ViFi.] {{\bf Example of fusion mRNA found by ViFi  \label{vifi_exmaple}}  We show an example of a fusion mRNA event found by ViFi that was not reported by the Tang et al. 2014 study nor found by VERSE for the sample TCGA-C5-A0TN on the genomic segment chr3:126,847,238-126,849,346.  The first track is the results from mapping the RNA-seq data, and the second track is the results from mapping the WGS data.  Reads in red are chimeric reads in which one end maps to the human hg19 reference (shown) and the other end maps to the viral reference.  The results show that both the top and bottom track contain chimeric reads clustered around this interval.}
%\end{figure}


\begin{figure}[htpb]
  \centering
  \includegraphics[width= 4in,frame]{{results/updown.250000}.pdf}
\caption[Fusion reads supp.]
{\label{updown_250kb}  {\bf Expression change around integration point within a 250kb window}.  We report the average log-fold expression change upstream and downstream of an integration event within a 250kb window.  The position is reported relative to the integration point in the human genome, with negative position being upstream of the integration event, and positive position being downstream of the integration event.  The results are separated out by the integration category. } %35, 110, 81
\end{figure}
\end{document}


\section*{Notes}

[1] N. Wentzensen, J. F. Coy, H. P. Knaebel, M. Linnebacher, B. Wilz, J. Gebert, and M. Von Knebel Doeberitz, “Expression of an endogenous retroviral sequence from the HERV-H group in gastrointestinal cancers,” Int. J. Cancer, vol. 121, no. 7, pp. 1417–1423, 2007.

Higher expression of HERV-H group in some cancers, not high in normal     

[1] M. T. Romanish, C. J. Cohen, and D. L. Mager, “Potential mechanisms of endogenous retroviral-mediated genomic instability in human cancer,” Semin. Cancer Biol., vol. 20, no. 4, pp. 246–253, 2010.

LTR and non-LTR not active, blocked by methalyation.  when meth. mech removed, the HERV become active.  

[1] A. Babaian and D. L. Mager, “Endogenous retroviral promoter exaptation in human cancer,” Mob. DNA, vol. 7, no. 1, p. 24, 2016.

LTR
Among them, pol is most conserved. Inside pol there are three important domains: IN(integrase), RT(reverse transcriptase) and RH(RNase H), which are enzymes for reverse transcription and insertion. RT and IN are regarded essential for autonomous LTR elements to fulfill their function.

LTR
no human LTRs transcriptionally active

Full length L1 have 2 internal promoters at 5' end, sense drives expression of element, antisense control transcription of nearby genes by forming chimeric transcripts, antisense promoter also express ORF0, which plays regulatory role in retrotransposition

500K L1, but only 3500-700 are full-length, retaining promoters




[1] M. Kadaja, A. Sumerina, T. Verst, M. Ojarand, E. Ustav, and M. Ustav, “Genomic instability of the host cell induced by the human papillomavirus replication machinery.,” EMBO J., vol. 26, no. 8, pp. 2180–91, 2007.

E1 protein is replicon origin recognition factor and viral helicase, which with E2, helps recruit(?) host cell replication complexes onto the HPV origin.  High E1 result in high recruitment of host replication complex at virus origin site.

Flanking DNA can be amplified, up to 12kb, with total amplicon size 25kb.  The amplification locus could be a target for genomic re-arrangments due to recruitment of DDR mechanisms.  In addition, onion-skin model of replication cuz breaks, fragments

[1] M. Kadaja, H. Isok-Paas, T. Laos, E. Ustav, and M. Ustav, “Mechanism of genomic instability in cells infected with the high-risk human papillomaviruses,” PLoS Pathog., vol. 5, no. 4, 2009.



[1] Z. Jiang, S. Jhunjhunwala, J. Liu, P. M. Haverty, M. I. Kennemer, Y. Guan, W. Lee, P. Carnevali, J. Stinson, S. Johnson, J. Diao, S. Yeung, A. Jubb, W. Ye, T. D. Wu, S. B. Kapadia, F. J. De Sauvage, R. C. Gentleman, H. M. Stern, S. Seshagiri, K. P. Pant, Z. Modrusan, D. G. Ballinger, and Z. Zhang, “The effects of hepatitis B virus integration into the genomes of hepatocellular carcinoma patients The effects of hepatitis B virus integration into the genomes of hepatocellular carcinoma patients,” pp. 593–601, 2012.

Runthrough for HBV, directional, need to make a clear graph
\section{Cell lines}
HPV16-positive W12 cell line both viral and integrated HPV genomes in early stages, lost of plasmids in later stages


%We examined the mean Fragments Per Kilobase of transcript per Million (FPKM) of genomic regions within 10kb of an integration compared to the mean FPKM of segments of samples without an integration in that region (Fig.~\ref{expression_transcripts}).  Genomic segments with integrations had increased expression of nearby human mRNA (2.5 fold increase), with significantly higher expression when human-viral fusion mRNA are present (5.5 fold increase).  Surprisingly, we found that integration also results in the expression of proximal LTR and LINE elements (3.5 to 5.3 fold increased expression when fusion mRNA is present).  


%and they have been implicated as causes in oncogenesis~\cite{Rodic2013,Xiao-Jie2016}.  While L1 elements are typically epigenetically repressed in  in germ and somatic cells,  , though this process is still poorly understood.  The integration process can create breaks in the host cell  genome, and it is hypothesized that and requires DNA repair mechanism to~\cite{Gasior2006,Rodic2013,Xiao-Jie2016}  
%
%LINE-1~\cite{Gasior2006}, and LINE L1 activity has been implicated in causing genomic instability and cancer~\cite{Rodic2013,Xiao-Jie2016}. 
%
%
%Hu et al. 2015 suggested that HPV may integrate randomly across the genome until an integration occurs in a region that provides a proliferation advantage and results in oncogenesis.  Thus, it's unsurprising to find that common regions with recurrent integrations are in or near known oncogenes. 
%
%~\cite{Tang2013,Hu2015}, and the second is identifying up-regulated or down-regulated host and viral genes resulting from integration~\cite{Tang2013}.  
%
% has highlighted genomic hotspots for integration, including known oncogenes such as MYC, PVT1, ERBB2, RAD51~\cite{Tang2013}, however, the vast majority of integrations are non-recurrent.  Hu et al. 2015 suggested that HPV may integrate randomly across the genome until an integration occurs in a region that provides a proliferation advantage and results in oncogenesis.  Thus, it's unsurprising to find that common regions with recurrent integrations are in or near known oncogenes.  
%
%Other research 
